api/types/client.go

```diff
@@ -166,7 +171,7 @@ func (s *SiteConfigSpec) getConsoleIngress() string {
 func ValidIngressOptions(platform Platform) []string {
 	switch platform {
 	case PlatformPodman:
-		return []string{}
+		return []string{IngressPodmanHost}
```

Comment:
Suggest considering ingress "none" option as it is plausible a user would not want to allow any connections to the site. Maybe distinguish site ingress from the local proxy edge connections coming into the site.

Reply:
Sure, I will include an ingress none option and if it is provided, it won't accept any `ingress-host`.